Outfitter Updates and Code Refactor (#10743)
endless-sky/endless-sky: data/_ui/tooltips.txt

# Shop Tips:
tip "outfitter: b"

> 	`Buy from outfitter and install on ship.`

Author: ```suggestion
	`_Buy from outfitter and install on ship.`
```

> 	`Buy from outfitter and install on ship.`

Reviewer: ```suggestion
	`Buy from the outfitter and install onto the currently selected ship(s).`
```

tip "outfitter: s"

> 	`Sell from cargo, storage or ship, in that order.`

Reviewer 2: ```suggestion
	`Sell from cargo, storage, or ship, in that order.`
```

> 	`Sell from cargo, storage, or ship, in that order.`

Author: ```suggestion
	`_Sell from cargo, storage, or ship, in that order.`
```

> 	`Sell from cargo, storage, or ship, in that order.`

Reviewer: ```suggestion
	`Sell from cargo, storage, or the currently selected ship(s), in that order.`
```

tip "outfitter: i"

> 	`Install on ship from cargo or storage, in that order.`

Author: ```suggestion
	`_Install on ship from cargo or storage, in that order.`
```

tip "outfitter: u"

> 	`Uninstall or move outfits from ship or cargo to storage, in that order.`

Author: ```suggestion
	`_Uninstall or move outfits from ship or cargo to storage, in that order.`
```

> 	`Uninstall from ship to storage or move outfits from cargo to storage, in that order.`

Reviewer 2: ```suggestion
	`Uninstall or move outfits from ship or cargo to storage, in that order.`
```

tip "outfitter: c"

> 	`Transfer outfit from planetary storage to cargo if possible, else buy from outfitter and place in cargo.`

Author: ```suggestion
	`Transfer outfit from planetary storage to _cargo if possible, else buy from outfitter and place in cargo.`
```

> 	`Transfer outfit from planetary storage to cargo if possible, else buy from outfitter and place in cargo.`

Reviewer 3: ```suggestion
	`Transfer outfit from planetary storage to cargo if possible, or buy from the outfitter.`
```

> 	`Transfer outfit from planetary storage to cargo if possible, else buy from outfitter and place in cargo.`

Author: ```suggestion
	`Transfer outfit from planetary storage to cargo if possible, otherwise buy it from the outfitter.`
```

> 	`Transfer to cargo from planetary storage if possible, else buy from outfitter and place in cargo.`

Reviewer 2: ```suggestion
	`Transfer outfit from planetary storage to cargo if possible, else buy from outfitter and place in cargo.`
```

tip "outfitter: r"

> 	`Move outfit from cargo or ship into planetary storage, in that order.`

Author: ```suggestion
	`Move outfit from cargo or ship into planetary sto_rage, in that order.`
```

> 	`Retain in storage from cargo if available, otherwise uninstall from ship to retain in storage.`

Reviewer 2: ```suggestion
	`Move outfit from cargo or ship into planetary storage, in that order.`
```
